### DISCUSSION TITLE
Post-beta release tagging and tracking

### DISCUSSION THREAD

**Discussion by wagnerck** [memtest86plus/memtest86plus#96] [Enhancement]
Great to see that beta 2 is out! That did remind me, though, that I'm not seeing anything on the memtest github related to release tagging or tracking. I'm wondering what the plans are for actual releases, since [we're looking into packaging the EFI executable and a support script](https://github.com/cwsystem76/pop-diagnostics) to allow Pop!OS users to almost-automatically add memtest as a boot option to their system without having to download and burn a USB stick. Pop!OS uses `systemd-boot` on UEFI systems, unlike Ubuntu and most other current distros, and that combined with memtest's default 64-bit Makefile defaults makes it nearly trivial to build a memtest EFI executable from source and add it to the ESP and the boot menu.

Once `memtest86plus` is out of beta and in actual stable release, are there going to be branches, tags, releases, or something similar that would make it easier for us to track point releases, and then update our package that uses this repo as a submodule? Having the beta releases tagged or tracked in some way would be helpful, but I totally understand if there are other priorities at the moment.

**Comment by x86fr** [comment]
Yes. All releases (including Beta) will be tagged soon.

PS: #95 merged

**Comment by x86fr** [comment]
Done for Beta2.

**Comment by wagnerck** [comment]
Awesome, thank you so much being on top of that.

And apologies for missing that issue, I probably should have looked deeper before asking.
